flexget/webserver.py

```diff
@@ -213,12 +219,22 @@ def _start_server(self):
             'log.screen': False,
         })
 
+        if self.ssl_certificate and self.ssl_private_key:
+            cherrypy.config.update({
+                'server.ssl_module': 'pyopenssl',
+                'server.ssl_certificate': self.ssl_certificate,
+                'server.ssl_private_key': self.ssl_private_key,
+            })
+
         try:
             host = self.bind if self.bind != "0.0.0.0" else socket.gethostbyname(socket.gethostname())
         except socket.gaierror:
             host = '127.0.0.1'
 
-        log.info('Web interface available at http://%s:%s' % (host, self.port))
+        if self.ssl_certificate and self.ssl_private_key:
+            log.info('Web interface available at https://%s:%s' % (host, self.port))
```

Comment:
Let the logger handle string formatting ie. `log.info('Web interface available at https://%s:%s' , host, self.port)`